### DISCUSSION TITLE
Consider adding support for bind on special network interface not just any address

### DISCUSSION THREAD

**Discussion by lifeengines** [LizardByte/community#565] [Sunshine] [answered]
### Select Topic Area

Feature Request

### Body

sometime time there are needs for bind on special network interface not only just any address,If possible maybe there can open multi-instances of sunshine to support mult-moniter stream easily.

**Comment by ReenigneArcher** [comment, marked as answer]
Added in https://github.com/LizardByte/Sunshine/pull/4481
